deploy/helm_charts/dc_website/README.md

```diff
@@ -0,0 +1,112 @@
+# Helm Charts
+
+The DC website can be installed into a GKE cluster using [helm](https://helm.sh/).
+
+## How to install the DC website on a live GKE cluster.
+
+### Prerequisites
+
+- A live GKE cluster with [workload identiy](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity) enabled. This is a one time setup.
+- [A global static IP reserved](https://cloud.google.com/compute/docs/ip-addresses/reserve-static-external-ip-address).
+- Domain from your choice of DNS provider, and have the a [DNS type A record](https://en.wikipedia.org/wiki/List_of_DNS_record_types) which points from the domain to the IP from above.
```

Comment:
Can we later investigate the cloud DNS as a default solution?

Reply:
As spoken offline, this will be explored after we get the setup workflow working with preallocated DNS and SSL setup.